CONTRIBUTING.md

```diff
@@ -62,9 +62,9 @@ In this case, it is appropriate to start the first line of a commit with '(doc)'
 
 + [Project Guidelines](https://logging.apache.org/log4j/2.x/guidelines.html)
 + [Code Style Guide](https://logging.apache.org/log4j/2.x/javastyle.html)
-+ [Apache Log4j 2 JIRA project page](https://issues.apache.org/jira/browse/LOG4J2)
++ [Apache Log4j 2 JIRA project page](https://issues.apache.org/jira/projects/LOG4J2/)
```

Comment:
For me, https://issues.apache.org/jira/projects/LOG4J2/ redirects to https://issues.apache.org/jira/projects/LOG4J2/issues

Reply:
I changed it right away.